webviz_core_components/wrapped_components/tabs.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any
+
+import dash_core_components as dcc
+
+
+class Tabs(dcc.Tabs):
+    """Returns a dcc.Tabs"""
+
+    def __init__(
+        self,
+        *args: Any,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(*args, **kwargs)
```

Comment:
What does it do compared with `dcc.Tabs`? To be implemented?

Reply:
Mainly added for ease of use. It is used together with the `Tab` component which is wrapped here with some styling.